content/github/getting-started-with-github/exploring-projects-on-github/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Exploring projects on GitHub
-intro: ''
+intro: 'Discover the relevant projects on {% data variables.product.prodname_dotcom %} and make contribution to Open Source by collaborating with other people.'
```

Comment:
This doesn't only apply to GitHub.com, so let's use the more general `product_name`, which will also render as GitHub Enterprise Server or GitHub AE as appropriate.

```suggestion
intro: 'Discover interesting projects on {% data variables.product.product_name %} and contribute to open source by collaborating with other people.'
```